README.md

```diff
@@ -1,2 +1,93 @@
 # code-clinic
 Our internal review for the packages we develop
+
+## Checklist
+
+#### Documentation
+
+The package includes all the following forms of documentation:
+
+- [ ] **A statement of need** clearly stating problems the software is designed to solve and its target audience in README.
+- [ ] **Installation instructions:** for the development version of the package and any non-standard dependencies in README.
+- [ ] **Vignette(s)** demonstrating major functionality that runs successfully locally.
+- [ ] **Function Documentation:** for all user-facing functions.
+- [ ] **Examples** for all user-facing functions.
+- [ ] **Community guidelines** including contribution guidelines in the README or CONTRIBUTING.
+- [ ] **Metadata** including author(s), author e-mail(s), a url, and any other relevant metadata e.g., in a `pyproject.toml` file or elsewhere.
+
+Readme file  requirements
+The package meets the readme requirements below:
+
+- [ ] Package has a README.md file in the root directory.
+
+The README should include, from top to bottom:
+
+- [ ] The package name
+- [ ] Badges for:
+    - [ ] Continuous integration and test coverage,
+    - [ ] Docs building (if you have a documentation website),
+    - [ ] A [repostatus.org](https://www.repostatus.org/) badge,
+    - [ ] Python versions supported,
+    - [ ] Current package version (on PyPI / Conda).
+
+*NOTE: If the README has many more badges, you might want to consider using a table for badges: [see this example](https://github.com/ropensci/drake). Such a table should be more wide than high. (Note that the a badge for pyOpenSci peer-review will be provided upon acceptance.)*
+
+- [ ] Short description of package goals.
+- [ ] Package installation instructions
+- [ ] Any additional setup required to use the package (authentication tokens, etc.)
+- [ ] Descriptive links to all vignettes. If the package is small, there may only be a need for one vignette which could be placed in the README.md file.
+    - [ ] Brief demonstration of package usage (as it makes sense - links to vignettes could also suffice here if package description is clear)
```

Comment:
I think this is important, and that's for the functions outlined there that I would like documentation (as I will likely use them as well)